packit/local_project.py

```diff
@@ -417,10 +417,10 @@ def checkout_pr(self, pr_id: Union[str, int]):
         self.git_repo.create_head(local_branch, f"{remote_name}/{local_branch}")
         self.git_repo.branches[local_branch].checkout()
 
-    def checkout_release(self, version: str) -> None:
-        logger.info(f"Checking out upstream version {version}.")
+    def checkout_release(self, upstream_tag: str) -> None:
```

Comment:
Does this need to be called `upstream_tag`? Would `tag` be enough?

Reply:
fixed